cds/modules/records/serializers/schemas/project.py

```diff
@@ -75,6 +74,7 @@ class ProjectSchema(StrictKeysSchema):
     title_translations = fields.Nested(TitleTranslationSchema, many=True)
     videos = fields.Nested(ProjectVideoSchema, many=True)
 
-    category = fields.Str()
-    type = fields.Str()
+    date = fields.Str()
+    category = fields.Str(required=True)
+    type = fields.Str(required=True)
```

Comment:
can you rename `type` to something more meaningful and less builtin, like `content_type`?